src/irmin-pack/unix/ext.ml

```diff
@@ -645,6 +647,7 @@ module Maker (Config : Conf.S) = struct
 
         let close process repo =
           flush repo;
```

Comment:
I thought about that path (couldn't find the code in tezos so thanks for that line!) but worried about slowing down imports with fsyncing every flush. Do you think it would be a problem?

Reply:
Ah, yes, you are right, sorry for the noise.

Reply:
Thanks for the confirmation. Not noise to me -- thanks for considering other approaches!